modules/oe_migration_workflow/tests/src/Unit/Plugin/migrate/process/SetWorkflowStateTest.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_migration_workflow\Unit\Plugin\migrate\process;
+
+use Drupal\Core\Config\ConfigManagerInterface;
+use Drupal\Core\Config\Entity\ConfigEntityTypeInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\oe_migration_workflow\Plugin\migrate\process\SetWorkflowState;
+use Drupal\migrate\Row;
+use Drupal\Tests\migrate\Unit\process\MigrateProcessTestCase;
+use Drupal\workflows\WorkflowInterface;
+use stdClass;
+
+/**
+ * @coversDefaultClass \Drupal\oe_migration_workflow\Plugin\migrate\process\SetWorkflowState
+ *
+ * @group oe_migration
+ */
+class SetWorkflowStateTest extends MigrateProcessTestCase {
```

Comment:
Missing a test case for `The entity status and the workflow state status don't match`

Reply:
True. Added the test case. I added it in a new test method because it's a different use case and the _expectException_ and _expectExceptionMessage_ calls are difficult to manage if we se it in the same method than other asserts.